test/PantosHub.t.sol

```diff
@@ -984,11 +986,49 @@ contract PantosHubTest is PantosHubDeployer {
 
     function test_registerToken() external {
         initializePantosHub();
-        mockPandasToken_getOwner(PANDAS_TOKEN_ADDRESS, deployer());
+        mockPandasToken_getOwner(PANDAS_TOKEN_ADDRESS, PANDAS_TOKEN_OWNER);
+        assertFalse(inArray(PANDAS_TOKEN_ADDRESS, loadPantosHubTokens()));
+        vm.expectEmit();
+        emit IPantosRegistry.TokenRegistered(PANDAS_TOKEN_ADDRESS);
+
+        vm.prank(PANDAS_TOKEN_OWNER);
+        pantosHubProxy.registerToken(PANDAS_TOKEN_ADDRESS);
+
+        PantosTypes.TokenRecord memory tokenRecord = pantosHubProxy
+            .getTokenRecord(PANDAS_TOKEN_ADDRESS);
+        assertTrue(tokenRecord.active);
+        assertTrue(inArray(PANDAS_TOKEN_ADDRESS, loadPantosHubTokens()));
+        checkTokenIndices();
+    }
+
+    function test_registerTokenBySuperCriticalOps() external {
+        initializePantosHub();
+        mockPandasToken_getOwner(PANDAS_TOKEN_ADDRESS, SUPER_CRITICAL_OPS);
+        assertFalse(inArray(PANDAS_TOKEN_ADDRESS, loadPantosHubTokens()));
+        vm.expectEmit();
+        emit IPantosRegistry.TokenRegistered(PANDAS_TOKEN_ADDRESS);
+
+        vm.prank(SUPER_CRITICAL_OPS);
+        pantosHubProxy.registerToken(PANDAS_TOKEN_ADDRESS);
+
+        PantosTypes.TokenRecord memory tokenRecord = pantosHubProxy
+            .getTokenRecord(PANDAS_TOKEN_ADDRESS);
+        assertTrue(tokenRecord.active);
+        assertTrue(inArray(PANDAS_TOKEN_ADDRESS, loadPantosHubTokens()));
+        checkTokenIndices();
+    }
+
+    function test_registerTokenBySuperCriticalOpsWhenPaused() external {
```

Comment:
Our naming convention would be `test_registerToken_BySuperCriticalOpsWhenPaused`.